scripts/ci-ecr

```diff
@@ -1,11 +1,11 @@
 dir=$(dirname "${BASH_SOURCE[0]}")
 GO_VERSION=$(cat "$dir/../GO_VERSION")
+GO_VERSION_WINDOWS=$(cat "$dir/../GO_VERSION_WINDOWS")
 
 IMAGES="registry:2.7.0
 golang:$GO_VERSION
+golang:$GO_VERSION_WINDOWS
 gcc:7.3.0
 busybox:1.32.0
-amazonlinux:2.0.20200722.0
-amazon/aws-for-fluent-bit:2.9.0
```

Comment:
do we need the ecr images here?
(Not clear on what this file does.)
```
- public.ecr.aws/amazonlinux/amazonlinux:2.0.20210126.0
- public.ecr.aws/aws-observability/aws-for-fluent-bit:2.10.1
```

Reply:
this mirrors some images from dockerhub to private ecr

Reply:
this file in particular is just sources by ci-ecr-pull and ci-ecr-push.....it's just a list of the images we need to mirror